### PR TITLE
helm lint support

### PR DESCRIPTION
**Pull request by dwi-di**

### CODE REVIEW

**Comment by dwi-di** [comment]
fixes #1

File: README.adoc [hunk 1]
```diff
@@ -83,9 +83,41 @@ that the correct docker image is used. An example snippet:
...
+|outputFile | target/test-classes/helm.yaml | output file for <<goal-template,template goal>>
 
 |===
 
+== Goals
+
+[#goal-package]
+=== Package
+
+goal packages a chart directory into a chart archive using the https://github.com/helm/helm/blob/master/docs/helm/helm_package.md[helm package] command.
```
Comment (pvorb) [inline README.adoc]: Could you start every sentence in the descriptions with a capital letter? Thanks.

**Comment by pvorb** [comment]
If you like, you can also add yourself to the [developers section](https://github.com/deviceinsight/helm-maven-plugin/blob/develop/pom.xml#L218-L243) in the `pom.xml`.

File: src/main/kotlin/com/deviceinsight/helm/LintMojo.kt [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+			val helm = resolveHelmBinary()
+
+			val params = ArrayList<String>()
+
+			if (strictLint) {
+				params.add("--strict")
+			}
+
+			if (valuesFile != null) {
+				params.add("--values ${project.basedir.resolve(valuesFile!!).absolutePath}")
```
[the remark this answers is not in the document]
Comment (dwi-di) [reply src/main/kotlin/com/deviceinsight/helm/LintMojo.kt]: actually I only copied it from `TemplateMojo`. Just removing the `!!` leads to a compile error. Fixing this  seems to exceed my kotlin skills :)

[the remark this answers is not in the document]
Comment (pvorb) [reply src/main/kotlin/com/deviceinsight/helm/LintMojo.kt]: :grinning:

**Pull request merged**